learning_resources/serializers_test.py

```diff
@@ -239,7 +240,10 @@ def test_learning_resource_serializer(  # noqa: PLR0913
         ],
         detail_key: detail_serializer_cls(instance=getattr(resource, detail_key)).data,
         "views": resource.views.count(),
-        "format": resource.format,
+        "learning_format": [
+            {"code": lr_format, "name": LearningResourceFormat[lr_format].value}
```

Comment:
The terminology is a bit confusing, for an `ExtendedEnum` class such as `LearningResourceFormat` , `name`  would be the code/slug (like `in_person`) and `value` would be the user-friendly word ("In person").  But for the serializers, we are  renaming the attributes: "code" instead of "name" and "name" instead of "value".  

Similar to here for the platform enum:
https://github.com/mitodl/mit-open/blob/b399a7db81bff29fd6d962b2a3b9b44fd5d53a23/learning_resources/serializers_test.py#L451-L453



The end result here needs to look like this (depending on the actual format):

```json
{
  "learning_format": [
            {"code": "in_person", "name": "In person"}
   ]
}
```